handler-proxy/src/main/java/io/netty/handler/proxy/HttpProxyHandler.java

```diff
@@ -108,32 +125,39 @@ public String password() {
     }
 
     @Override
-    protected void addCodec(ChannelHandlerContext ctx) throws Exception {
+    protected void addCodec(ChannelHandlerContext ctx) {
```

Comment:
@slandelle you need to revert all the removals of `throws...` as its a breaking change. People who may have override the method and have `throws` on it will be in trouble.